tests/events_test.py

```diff
@@ -29,6 +30,7 @@ def testEventMask(self):
         device = "sdc"
         action = "add"
         mgr.handle_event(action, device)
+        time.sleep(1)
```

Comment:
These are not py2-specific. The failure was sporadic, only occurring maybe three times out of 25 or so runs, but it made me aware of the potential for a race.